tests/test_update_manager_file_cache.py

```
@@ -0,0 +1,448 @@
+        finally:
+            conn.close()
+
+        # Test cache data retrieval
+        cache_data = await manager._get_cache_data_for_checksum("openfoodfacts")
+        assert isinstance(cache_data, dict)
+        assert "apple" in cache_data
+        assert "banana" in cache_data
+        assert "checksum" in cache_data["apple"]
+
+    @pytest.mark.asyncio
+    async def test_usda_update_success_path_coverage(self, manager, temp_dir):
```

Ruff (0.14.1)

Unused method argument: `temp_dir`

(ARG002)